src/main/java/teammates/ui/template/FeedbackResponseCommentRow.java

```diff
@@ -320,6 +320,9 @@ private String getTypeOfPeopleCanViewComment(FeedbackResponseCommentAttributes c
             case INSTRUCTORS:
                 peopleCanView.append("instructors, ");
                 break;
+            case OWN_TEAM_MEMBERS_INCLUDING_SELF:
+                peopleCanView.append("response giver's team members and response giver, ");
+                break;
             default:
                 break;
```

Comment:
Those html changes were because all valid viewers were not covered in this method. Fixed it.

Reply:
If we miss the viewer, why it doesn't happen before?